ipatests/test_webui/test_automember.py

```diff
@@ -251,7 +254,7 @@ def test_crud(self):
         self.delete(hostgroup.ENTITY, [hostgroup.DATA])
 
     @screenshot
-    def test_rebuild_membership_hosts(self):
+    def test_rebuild_membership_hosts(self, confirm_btn="ok"):
```

Comment:
No need to add confirm_btn as a parameter of the test function (the function is never called with a different value for confirm_btn).

Reply:
So now, you can see that as soon as I removed the `confirm_btn` as a parameter then the `AzurePipeline.freeipa (Lint)` check started failing and gives `Undefined variable 'confirm_btn'` error.

Reply:
You need to replace confirm_btn in lines 293, 320, 366, and 393 with 'ok":
`self.dialog_button_click(confirm_btn)` -> `self.dialog_button_click('ok')`